Track Buddy/MotionManager.swift

```diff
@@ -66,12 +69,57 @@ class MotionManager: ObservableObject {
     private var recentPoints: Deque<CGPoint> = [] // TODO: dluo- think about thread safety?
     
     func pointPath(atScale factor: CGFloat) -> CGMutablePath {
```

Comment:
Typical naming would be `pointPath(scaleFactor: CGFloat)` or `pointPath(with scaleFactor: CGFloat)`